plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -2,13 +2,332 @@
+            };
+        },
+        computed: {
+        },
+        props: {
+            value: String,
+            isOpen: Boolean
+        },
+        watch: {
+            value: {
+                immediate: true,
+                handler: function(newValue) {
+                    this.currentVal = newValue;
+                }
+            },
+        },
+        methods: {
+            handleClose: function() {
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditor", false);
+                this.$store.dispatch("countlyRemoteConfig/parameters/showJsonEditorForCondition", false);
+                this.$emit("input", this.currentVal);
+            }
+        }
+    });
+    var ConditionDialog = countlyVue.views.create({
+        template: CV.T("/remote-config/templates/condition-dialog.html"),
+        mixins: [
+            _mixins.MultiStepForm,
+        ],
+        data: function() {
+            var additionalProperties = [];
+            var remoteConfigFilterRules = [new countlyQueryBuilder.RowRule({
+                name: "cly.remote-config-random-percentile-rule",
+                selector: function(row) {
+                    return row.property.id === "up.random_percentile";
+                },
+                actions: [new countlyQueryBuilder.RowAction({
+                    id: "disallowOperator",
+                    params: {
+                        selector: function(operator) {
+                            return ["cly.=", "cly.!=", "cly.contains", "cly.between", "cly.isset"].includes(operator.id);
+                        }
+                    }
+                })]
+            })];
+            additionalProperties.push(new countlyQueryBuilder.Property({
+                id: 'up.random_percentile',
+                name: CV.i18n("remote-config.conditions.random.percentile"),
+                type: countlyQueryBuilder.PropertyType.NUMBER,
+                group: 'User Properties',
+
+            }));
+            return {
+                selectedTag: {},
+                conditionDialog: {
+                    condition_name: "",
+                    condition_color: 1,
+                    condition: {},
+                    condition_definition: "",
+                    seed_value: "",
+                    condition_description: ""
+                },
+                remoteConfigFilterRules: remoteConfigFilterRules,
+                useDescription: false,
+                managedPropertySegmentation: {},
+                conditionPropertySegmentation: { query: {}, byVal: []},
+                additionalProperties: additionalProperties,
+                defaultTag: {
+                    value: 1,
+                    label: "#6C47FF"
+                },
+                colorTag: COLOR_TAG
+            };
+        },
+        computed: {
+            isOpen: function() {
+                return this.$store.getters["countlyRemoteConfig/parameters/showConditionDialog"];
```

Comment:
why add `showConditionDialog` in vuex?
won't keeping the dialogs show state in the components local state work?